src/main/java/paymelah/model/debt/DebtList.java

```diff
@@ -26,6 +28,7 @@ public class DebtList {
     public DebtList(DebtList debtList) {
         requireAllNonNull(debtList);
         debts.addAll(debtList.debts);
+        totalDebt = totalDebt.add(debtList.totalDebt);
```

Comment:
Should the other `DebtList`'s `totalDebt` just be copied over (instead of making a new `BigDecimal` by adding 0 to it)?